### PR TITLE
Refactor capture into separate class, remove old client

### PR DESCRIPTION
**Pull request by pgesek**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/m2sysbiometrics/capture/impl/CloudScanrCaptor.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    private Token getToken() {
+        String username = properties.getCloudScanrUsername();
+        String password = properties.getCloudScanrPassword();
+        String customerKey = properties.getCustomerKey();
+        String cloudScanrUrl = properties.getCloudScanrUrl();
+
+        return httpClient.getToken(cloudScanrUrl, username, password, customerKey);
+    }
```
Comment (dserkowski) [inline api/src/main/java/org/openmrs/module/m2sysbiometrics/capture/impl/CloudScanrCaptor.java:48]: The code for getToken duplicated - I noticed it also in the AbstractM2SysClient class. Should it be moved somewhere, for example to a Util class or M2SysHttpClient?

**Review by dserkowski — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/m2sysbiometrics/capture/impl/CloudScanrCaptor.java (hunk 1 above)
Comment (pgesek) [reply api/src/main/java/org/openmrs/module/m2sysbiometrics/capture/impl/CloudScanrCaptor.java:48]: I think we can move it once it duplicates three times, following the rule of three

Comment (dserkowski) [reply api/src/main/java/org/openmrs/module/m2sysbiometrics/capture/impl/CloudScanrCaptor.java:48]: Ok

**Comment by dserkowski** [comment]
LGTM

**Review by dserkowski — APPROVED** [review]

**Pull request merged**